pixi.toml

```diff
@@ -87,18 +87,18 @@ geopandas = "*"
 jupyterlab = "*"
 matplotlib = "*"
 mypy = "*"
-pandas = "<2.1"
+pandas = "2.1.2.*"
 pandas-stubs = "*"
 pandera = "*"
 pip = "*"
 pre-commit = "*"
 pyarrow = "*"
 pydantic = "~=1.0"
-pyogrio = "*"
+pyogrio = "0.7.2.*"
```

Comment:
Why do we need to pin pyogrio?
It should just take the newest, no?

Reply:
I just restored this. `pixi add` writes this by default, but good to keep this open and only write down real version requirements.

Reply:
Yeah, not a big fan of the default behaviour of `pixi add`.
It makes kind of sense for `cargo add`, but no too much in the conda ecosystem IMO.
They disagree with me though when I opened an issue for that :P